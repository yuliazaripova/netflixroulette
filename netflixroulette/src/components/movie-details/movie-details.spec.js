import * as React from 'react';

import MovieDetails from './movie-details';

describe("MovieDetails component", () => {
  it("should render MovieDetails component", () => {
    const testProps = {
        id: 1,
        title: "test title",
        vote_average: 1,
        release_date:  "2017-12-13",
        poster_path: "test poster",
        overview: "test overview",
        runtime: 1,
        genres: ["test genre1", "test genre2"],
    }
    const component = shallow(<MovieDetails {...testProps}/>);
    expect(component).toMatchSnapshot();
  })
})