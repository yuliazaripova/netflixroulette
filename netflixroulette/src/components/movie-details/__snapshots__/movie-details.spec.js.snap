// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetails component should render MovieDetails component 1`] = `
<div
  className="movie-details"
>
  <Poster
    classes="movie-details_poster"
    poster_path="test poster"
    title="test title"
  />
  <div
    className="movie-details_details"
  >
    <h2
      className="movie-details_title"
    >
      test title
    </h2>
    <span
      className="movie-details_vote"
    >
      1
    </span>
    <p>
      <span
        className="movie-details_date"
      >
        2017
      </span>
      <span
        className="movie-details_runtime"
      >
        1
         min
      </span>
    </p>
    <p
      className="movie-details_overview"
    >
      test overview
    </p>
  </div>
</div>
`;
